2a_model.R

```diff
@@ -91,26 +96,31 @@ p2a_targets_list <- list(
     },
     format="file"
   ),
-
+  
+ 
+  
   # gather model ids - add to this list when you want to reproduce
-  # outputs from a new model
+  # outputs from a new model #add medium observed sites
   tar_target(
     p2a_model_ids,
     # paths are relative to 2a_model/src/models
     list(list(model_id = "0_baseline_LSTM",
-              snakefile_dir = "0_baseline_LSTM",
-              config_path = "0_baseline_LSTM/config.yml"),
+               snakefile_dir = "0_baseline_LSTM",
+               config_path = "0_baseline_LSTM/config.yml"),
          # the 1_ models use the same model and therefore
          # the same Snakefile as the 0_baseline_LSTM run
-         list(model_id = "1_metab_multitask",
+         # list(model_id = "1_metab_multitask",
```

Comment:
Why did you comment these out? Did you try them and they didn't work?

Reply:
Ah, just so I could specifically run the baseline models, I will uncomment them